src/services/api.ts

```diff
@@ -111,9 +96,9 @@ export const loginUser = async (
 ) => {
   try {
     const res = await instance.post("/api/auth/login", {
-      username: username,
-      password: password,
-      email: email,
+      username,
+      password,
+      email,
     });
     const { accessToken, refreshToken } = res.data;
     localStorage.setItem("accessToken", accessToken);
```

Comment:
remove that. Now we use redux
and function loginUserAsLeleka1 change: delete  localStorage usage

Reply:
this functions is only for test reasons and can be completelly removed

Reply:
Thanks, I was just about to add)